src/Admin/FieldDescription.php

```diff
@@ -35,6 +30,7 @@ public function setAssociationMapping($associationMapping)
 
         $this->type = $this->type ?: $associationMapping['type'];
         $this->mappingType = $this->mappingType ?: $associationMapping['type'];
+        // NEXT_MAJOR: Remove the next line.
```

Comment:
The next line is uselss because the fieldName is supposed to be set to find the association mapping.
Plus in next major, the field name will be set with `new FieldDescription('FieldName')`